src/cloudsc_gpu/CMakeLists.txt

```diff
@@ -170,6 +171,21 @@ if( HAVE_CLOUDSC_GPU_SCC_CUF )
 endif()
 
 
+if( HAVE_CLOUDSC_GPU_SCC_FIELD )
+    ecbuild_add_executable(
+        TARGET dwarf-cloudsc-gpu-scc-field
+        SOURCES
+            dwarf_cloudsc_gpu.F90
+            cloudsc_driver_gpu_scc_field_mod.F90
+            cloudsc_gpu_scc_mod.F90
+        LIBS
+            cloudsc-common-lib
+        DEFINITIONS ${CLOUDSC_DEFINITIONS} CLOUDSC_GPU_SCC_FIELD
+    )
+    set_target_properties(dwarf-cloudsc-gpu-scc-field PROPERTIES LINK_FLAGS "-cuda -gpu=pinned")
```

Comment:
Use `target_link_options` to add the `-gpu=pinned` flag. `-cuda` should be provided by the options set on the common lib.